Fix #374

oko opened the pull request:
SQLAlchemy doesn't have the module `sqlalchemy.sql.schema` until v0.9 (imported in `flask_restless.views`).

Quick PR to bump `requirements.txt` and the README to v0.9.

Fixes #374

jfinkels commented:
Instead of increasing version requirements, why don't we just change

    from sqlalchemy.sql.schema import Column

to
    from sqlalchemy import Column

(Sidenote: all our imports should probably be of this simpler form in order to avoid problems like this in the future.)

oko commented:
Reverted the original commit and moved the import.

The import of `sqlalchemy.sql.func` on line 29 in [`flask_restless/helpers.py`](https://github.com/jfinkels/flask-restless/blob/4bcd34fcffd43e139f4de046926e7a1eec9fb270/flask_restless/helpers.py#L29) is also importable from `sqlalchemy` directly. Change or leave?

jfinkels commented:
Leave it for now, thanks.
